source/fundamentals/aggregation.txt

```diff
@@ -0,0 +1,220 @@
+
+The following table lists the different tasks you can perform with find
+operations, compared to what you can achieve with aggregation
+operations. The aggregation framework provides expanded functionality
+that allows you to transform and manipulate your data.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Find Operations
+     - Aggregation Operations
+
+   * - | Select *certain* documents to return
+       | Select *which* fields to return
+       | Sort the results
+       | Limit the results
+       | Count the results
+     - | Select *certain* documents to return
+       | Select *which* fields to return
+       | Sort the results
+       | Limit the results
+       | Count the results
+       | Rename fields
+       | Compute new fields
+       | Summarize data
+       | Connect and merge data sets
+
+Server Limitations
+------------------
+
+Consider the following :manual:`limitations </core/aggregation-pipeline-limits/>` when 
+performing aggregation operations:
+
+- Returned documents must not violate the :manual:`BSON document size limit </reference/limits/#mongodb-limit-BSON-Document-Size>`
+  of 16 megabytes.
+
+- Pipeline stages have a memory limit of 100 megabytes by default. If required, you can exceed this limit by setting
+  the `AllowDiskUse <{+new-api-root+}/MongoDB.Driver/MongoDB.Driver.AggregateOptions.AllowDiskUse.html#MongoDB_Driver_AggregateOptions_AllowDiskUse>`__
+  property of the ``AggregateOptions`` object that you pass to the ``Aggregate()`` method.
+
+- The :manual:`$graphLookup </reference/operator/aggregation/graphLookup/>` stage has
+  a strict memory limit of 100 megabytes and ignores the ``AllowDiskUse`` property.
+
+Aggregation Example
+-------------------
+
+.. note::
+  
+  This example uses the ``sample_restaurants.restaurants`` collection
+  from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+  free MongoDB Atlas cluster and load the sample datasets, see :ref:`csharp-quickstart`.
+
+To perform an aggregation, pass a list of aggregation stages to the
+``IMongoCollection<TDocument>.Aggregate()`` method.
+
+The following code example produces a count of the number of bakeries in each borough
+of New York. To do so, it uses an aggregation pipeline with the following stages:
+
+- A :manual:`$match </reference/operator/aggregation/match/>` stage to filter for documents whose
+  ``cuisine`` field contains the value ``"Bakery"``.
+
+- A :manual:`$group </reference/operator/aggregation/group/>` stage to group the matching
+  documents by the ``borough`` field, accumulating a count of documents for each distinct value
+  of that field.
+
+We have provided three versions of this example that each provide different approaches to
+defining the aggregation stages used in the example pipeline. The first uses 
+``BsonDocument`` objects, the second uses LINQ, and the third uses builders.
+
+.. tabs::
+
+   .. tab:: ``BsonDocument`` approach
+      :tabid: bsondocument
```

Comment:
S: remove monospacing from header elements, Also I think that this tab should be last given the technical feedback
```suggestion
   .. tab:: BsonDocument
      :tabid: bsondocument approach
```

Reply:
same formatting suggestion applies to all three examples